src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/WalletForm.tsx

```diff
@@ -80,7 +80,9 @@ export const WalletForm = ({ signatories, onContinue, isActive, isLoading, onGoB
     );
   });
 
-  const accountAlreadyExists = accounts.some((a) => a.accountId === multisigAccountId);
```

Comment:
Can't we do `.some(a => !accountUtils.isProxiedAccount(a) && a.accountId === multisigAccountId)`?